app/js/services/resultatService.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-angular.module('ddsApp').service('ResultatService', function($http, droitsDescription) {
+angular.module('ddsApp').service('ResultatService', function($http, $modal, droitsDescription) {
```

Comment:
It's an inner “popup” that blocks the view. [Ref](https://fr.wikipedia.org/wiki/Fenêtre_modale).